nevergrad/functions/irrigation/irrigation.py

```diff
@@ -55,8 +55,10 @@ def __init__(self, symmetry: int, n_iterations: int = 1000) -> None:
         if os.environ.get("CIRCLECI", False):
             raise ng.errors.UnsupportedExperiment("No HTTP request in CircleCI")
 
+        
+        
         self.cropd = YAMLCropDataProvider(
-            repository="https://raw.githubusercontent.com/ajwdewit/WOFOST_crop_parameters/master/"
+            repository= "https://github.com/ajwdewit/pcse_notebooks/blob/master/data/crop/SUG0601.crop"
```

Comment:
@Ethel2003 you shouldn't change this part. I suggest you drop this change: it won't work.

Reply:
@akouete-kpakpo I did this changement because I thaught that the link was specify for the culture of rice.

Reply:
yes @Ethel2003 but as you can see there is no mention of rice in "https://raw.githubusercontent.com/ajwdewit/WOFOST_crop_parameters/master/"

Reply:
okay, I did all the modifications and  did push then